docs/topics/whatsnew-eap.md

```diff
@@ -9,7 +9,7 @@ _[Release date: %kotlinEapReleaseDate%](eap.md#build-details)_
 
 The Kotlin 1.7.20-RC release is out! Here are some highlights from this preview version of Kotlin:
 
-* [The new Kotlin K2 compiler supports `all-open`, SAM with receiver, Lombok, Parcelize, and other compiler plugins](#support-for-kotlin-k2-compiler-plugins)
+* [The new Kotlin K2 compiler supports `all-open`, `no-arg`, SAM with receiver, Lombok, Parcelize, and other compiler plugins](#support-for-kotlin-k2-compiler-plugins)
```

Comment:
JFYI:
`no-arg` was removed here: https://github.com/JetBrains/kotlin-web-site/commit/107c28331ae39ce55245ffa44e4199913dfcac0c#diff-7965e4f6aaaea3be50bb068bf47c0e2e56f67f0ea7d4caf2697303362874e72dL12-R12

Looks like it's a mistake, so I added it back

Reply:
OK
Thanks for the information